tests/bootstrap.php

```diff
@@ -94,7 +89,7 @@ class PHP_Token_FN extends PHP_Token
 ConnectionManager::alias('test', 'default');
 
 Router::reload();
-Security::setSalt('BEDITA');
+Security::setSalt('BEDITA|BEDITA|BEDITA|BEDITA|BEDITA|BEDITA|BEDITA|BEDITA');
```

Comment:
Questo non ho idea di cosa sia, forse lo ha messo il php-cs fix automatico. Rimuoviamo o è giusto così? @edoardocavazza @fquffio

Reply:
No, I think it's due to the newly introduced requirement of a minimum size in bytes for the security salt. Although the value we set here is bogus for the sole purpose of unit tests, it still has to fulfil that requirement.

Reply:
Makes sense, thanks fquffio.